Cargo.lock

```diff
@@ -133,6 +153,12 @@ version = "3.8.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "8f1e260c3a9040a7c19a12468758f4c16f31a81a1fe087482be9570ec864bb6c"
 
+[[package]]
+name = "cc"
```

Comment:
hmm..

Reply:
hmm indeed... native C code is a necessary dependency? 😕

Reply:
Look like it comes from [backtrace](https://crates.io/crates/backtrace), surprising given that backtrace is an "official" crate...

Reply:
Looks like it is only used when targetting android, there is a single .c binding file.  There is this note in the [Cargo.toml](https://github.com/rust-lang/backtrace-rs/blob/master/Cargo.toml#L53-L56):

```toml
[build-dependencies]
# Only needed for Android, but cannot be target dependent
# https://github.com/rust-lang/cargo/issues/4932
cc = "1.0.67"
```

Reply:
Opened https://github.com/zkat/miette/issues/132